### DISCUSSION TITLE
What are some underrated Android features that users should try?

### DISCUSSION THREAD

**Discussion by viralsweet** [community/community#147957] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

What are some cool but underrated Android Features on Android that most people don’t know about? those little things that make life easier but don’t get talked about much. I feel like there’s so much hidden stuff in Android that we miss out on.

**Comment by abjodas** [comment]
Well, it depends on the particular skin you are using, but some of the most useful features that I have found are as follows:
1)Split-Screen Mode
2)Screen Pinning
3)Guest Mode
4)App-Specific Volume Control
5)Lockdown Mode
6)Gesture Navigation Tweaks
7)Live Transcribe and Captioning
8)Notification History
9)Developer Options Goodies
10)Digital Wellbeing Features
11)Custom App Shortcuts
12)One-Handed Mode
13)Power Button Double Tap for Camera

**Reply by viralsweet** [reply]
Some of these are really cool Like split screen mode I use that sometimes and it’s super handy, especially for YouTube and browsing together. And live transcribe and captions are awesome didn’t even know about that till recently! 
But idk about stuff like notification history I feel like I never needed that? Maybe it’s just me lol. And the developer options… isn’t that more for techy people? Feels kinda complicated for regular use. Also, one handed mode sounds useful but I always forget it’s there when I actually need it. 😂 Still Android really does hide some good stuff!

**Comment by lisamarcus7788** [comment]
There are so many underrated Android features that people don’t talk about enough One cool feature I’ve been loving lately is Glance. It’s like this super handy lock screen experience that gives you quick updates on stuff you actually care about without even unlocking your phone.
You can get everything from news and sports scores to fun facts and trends, all at a glance (pun intended). It’s customizable too, so it feels super personal to your interests. Honestly it’s one of those Android features that’s simple but makes your day-to-day life way easier.
